### PR TITLE
Display question and swap question in collaboration rooms

### LINKED ISSUE #80: Matching Service - Should handle passing the question to FE
Pass by question ID

### LINKED ISSUE #149: Frontend- Matching Service-Collaboration: Fetch and display question in collaboration service
Await #80 

?room&question

### LINKED ISSUE #216: Frontend - Matching - Question Service: Swap question functionality in collaboration rooms

### PR DESCRIPTION
**Pull request by gycgabriel**
Fix #80, fix #149, fix #216

### CODE REVIEW

**Review by ong6 — APPROVED** [review]
Lgtm

File: frontend/src/pages/room/[id].tsx [hunk 1]
```diff
@@ -12,6 +12,7 @@ export default function Room() {
   const router = useRouter();
 
   const roomId = router.query.id as string;
+  const questionId = router.query.questionId as string;
```
Comment (ong6) [inline frontend/src/pages/room/[id].tsx]: use db to fetch instead

**Comment by gycgabriel** [comment]
IT WORKS now@!!!

image

**Comment by gycgabriel** [comment]
Added swap question functionality, can click button to swap questions

--> Updates questionId in match + collab service

**Pull request merged**